azure-pipelines-dependency-graph-testing.yml

```diff
@@ -602,9 +597,7 @@ stages:
 
           - script: |
               # Install dependencies
-              pip install pytest
```

Comment:
I think you accidentally removed `pytest` here.

Reply:
should be good since it is in `dgtest` `install_requires`